packages/theming/theme-tokens/src/getColorTokens.android.ts

```diff
@@ -0,0 +1,12 @@
+import lightAliasTokens from '@fluentui-react-native/design-tokens-android/light/tokens-aliases.json';
```

Comment:
This package should be global-tokens only - alias tokens should be dealt with in their own platform-theme packages.

Reply:
theme-tokens is a bit of a misnomer since it used to hold all the tokens, but we moved alias tokens out of this package based on feedback from clients.